cmake/BuildOptionDefaults.cmake

```diff
@@ -5,7 +5,7 @@
 # and not on the command line.
 # ~~~
 set(NRN_ENABLE_SHARED_DEFAULT ON)
-set(NRN_ENABLE_INTERVIEWS_DEFAULT ON)
+set(NRN_ENABLE_INTERVIEWS_DEFAULT OFF)
```

Comment:
Actually, this is more the opposite of an autotools build in which the absence of --with-iv is equivalent to its presence.
The autotools build goes through a sequence of locations in which to find an interviews installation. Nevertheless,
I have no objection to requiring the  option for cmake if one wants to enable InterViews

Reply:
Ah yes! I will prefer to change this then.